README.md

```diff
@@ -28,15 +30,15 @@ registers_client = RegistersClient::RegisterClientManager.new({
 })
 ```
 
-The `RegisterClientManager` maintains individual instances of `RegisterClient` for each register you access. 
+The `RegisterClientManager` maintains an invidual instance of [`RegisterClient`](#registerclient) for each time you access a register by calling [`get_register`](#getregister).
+
+`get_register` is the only public method available on `RegisterClientManager`.
```

Comment:
This isn't strictly true any longer after Michaela's recent PR - there is now an additional method `get_register_from_environment`. I will document this separately.

Reply:
OK, which part is no longer true? Or both?

Reply:
The first part (line 33) is true in my proposed comment prior to this. `get_register` being the only public method available on `RegisterClientManager` was true, but is not any longer, as an additional public method `get_register_from_environment` has been added.

Reply:
OK, I've removed line 35. Thanks @karlbaker02